src/devices/Bmp180/Bmp180.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <ApplicationIcon />

```

Comment:
NIT: ApplicationIcon and StartupObject are not used so they should be removed. Also, OutputType defaults to Library so that one is not required either. Same goes for PlatformTarget.

Reply:
NIT = Not important thing 😄

Reply:
I always thought nit was short for nit-pick:
https://stackoverflow.com/questions/27810522/what-does-nit-mean-in-hacker-speak

One of the suggestions was (N)ot (I)mportant, (T)hough :)